### ISSUE TITLE
EntityDataView.refreshData should consider current filter

### ISSUE THREAD

**Issue by korzh** [KorzhCom/EasyData#131]
When we update the grid after the data fetch we should consider the current filter as well.

**Comment by antifree** [comment]
Yep. I wanted to refactor in such a case that's why fixing #127 took more time. 
And have you checked that record after the update is displayed correctly in you fix?

**Comment by korzh** [comment]
Fixed in [2b73d44](https://github.com/KorzhCom/EasyData/commit/2b73d44abe555a75311b91e79a2f7ed489676fe6)
